write-api-client/src/main/java/io.dockstore.client/cli/Add.java

```diff
@@ -0,0 +1,176 @@
+package io.dockstore.client.cli;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import io.swagger.client.write.ApiException;
+import io.swagger.client.write.api.GAGHoptionalwriteApi;
+import io.swagger.client.write.model.Tool;
+import io.swagger.client.write.model.ToolDescriptor;
+import io.swagger.client.write.model.ToolDockerfile;
+import io.swagger.client.write.model.ToolVersion;
+import json.Output;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static io.dockstore.client.cli.ConfigFileHelper.getIniConfiguration;
+import static io.dockstore.client.cli.ExceptionHelper.CLIENT_ERROR;
+import static io.dockstore.client.cli.ExceptionHelper.errorMessage;
+
+/**
+ * @author gluu
+ * @since 23/03/17
+ */
+class Add {
+    // an organization for both GitHub and Quay.io where repos will be created (and deleted)
+    private static final Properties PROPERTIES = getIniConfiguration();
+    private static final String ORGANIZATION_NAME = PROPERTIES.getProperty("organization", "dockstore-testing");
```

Comment:
Your main readme should document all available configuration parameters and their defaults.